tox.ini

```diff
@@ -154,7 +154,7 @@ deps=
 [testenv:func-{,grpc-,docs-,noml-,}py{37,38,39,310}]
 install_command=pip install --extra-index-url https://download.pytorch.org/whl/cpu {opts} {packages}
 deps =
-    yea-wandb=={env:YEA_WANDB_VERSION}
+    https://github.com/wandb/yea-wandb/archive/create-launch-agent-fields.zip
```

Comment:
dev tracks? seems like this should not be committed to main

Reply:
Following along with the [functional testing guide](https://github.com/wandb/wandb/blob/main/CONTRIBUTING.md#functional-testing), once this PR is approved and ready to ship, I'll merge my [yea-wandb change](https://github.com/wandb/yea-wandb/pull/118), change this to use the new yea-wandb version, then merge this change.

Reply:
ah thanks, TIL